docs/release_notes.md

```diff
@@ -5,6 +5,102 @@ hide:
+- [[PR #11050]](https://github.com/kubevirt/kubevirt/pull/11050) [fossedihelm] restrict default cluster role to authenticated only users
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
+- [[PR #10879]](https://github.com/kubevirt/kubevirt/pull/10879) [brianmcarey] Built with golang 1.20.12
+- [[PR #10863]](https://github.com/kubevirt/kubevirt/pull/10863) [dhiller] Remove year from generated code copyright
+- [[PR #10787]](https://github.com/kubevirt/kubevirt/pull/10787) [matthewei] virtctl support to add template label and annotation filters
+- [[PR #10720]](https://github.com/kubevirt/kubevirt/pull/10720) [awels] Restored hotplug attachment pod request/limit to original value
+- [[PR #10637]](https://github.com/kubevirt/kubevirt/pull/10637) [dharmit] Functional tests for sidecar hook with ConfigMap
+- [[PR #10615]](https://github.com/kubevirt/kubevirt/pull/10615) [orelmisan] Remove leftover NonRoot feature gate
+- [[PR #10598]](https://github.com/kubevirt/kubevirt/pull/10598) [alicefr] Add PVC option to the hook sidecars for supplying additional debugging tools
+- [[PR #10596]](https://github.com/kubevirt/kubevirt/pull/10596) [mhenriks] Disable HTTP/2 to mitigate CVE-2023-44487
+- [[PR #10582]](https://github.com/kubevirt/kubevirt/pull/10582) [orelmisan] Remove leftover NonRootExperimental feature gate
+- [[PR #10567]](https://github.com/kubevirt/kubevirt/pull/10567) [awels] Attachment pod creation is now rate limited
+- [[PR #10526]](https://github.com/kubevirt/kubevirt/pull/10526) [cfilleke]  Documents steps to build the KubeVirt builder container
+- [[PR #10479]](https://github.com/kubevirt/kubevirt/pull/10479) [dharmit] Ability to run scripts through hook sidecardevice
+- [[PR #10244]](https://github.com/kubevirt/kubevirt/pull/10244) [hshitomi] Added “adm” subcommand under “virtctl”, and “log-verbosity" subcommand under “adm”. The log-verbosity command is: to show the log verbosity of one or more components, to set the log verbosity of one or more components, and to reset the log verbosity of all components (reset to the default verbosity (2)).
+- [[PR #10046]](https://github.com/kubevirt/kubevirt/pull/10046) [victortoso] Add v1alpha3 for hooks and fix migration when using sidecars
```

Comment:
bugfix?

Reply:
Alice previously suggested moving this out of bugfix as it also adds alpha1